hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -228,8 +243,20 @@ public Void call() throws Exception {
       init(freon.createOzoneConfiguration());
     }
 
-    keyValue =
-        DFSUtil.string2Bytes(RandomStringUtils.randomAscii(keySize - 36));
+    keyValueBuffer = DFSUtil.string2Bytes(
+        RandomStringUtils.randomAscii(bufferSize));
```

Comment:
Let's just write zeroes by default. Random data generation may itself become a bottleneck. We could add. a separate option for random data later.

Reply:
Thank you, but the generation of random data may not take too much time, just like xiaoyuyao said below, repeatedly calculate the same content won't change the digest, so as an optimization, MessageDigest.update() is only required for the first calculation and the last calculation. this is quickly indeed.